### PR TITLE
(Fixes #11464) Set up variation page for focus page

### LINKED ISSUE #11464: Create variation of focus page for product test
## Description

Product team will be driving traffic to the firefox focus page. They have requested a page variation so that we can have better attribution for this test.

original page: `/firefox/browsers/mobile/focus/`
Page variation URL: `/firefox/browsers/mobile/focus/?v=pb`

Google Play Button URL: `https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus-omc171-uk&redirect=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dorg.mozilla.focus`

App Store Button URL: `https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus
qr-code
-omc171-uk&redirect=https%3A%2F%2Fapps.apple.com%2Fus%2Fapp%2Ffirefox-focus-privacy-browser%2Fid1055677337`

QR Code: attached
qr-code

---

### PR DESCRIPTION
**Pull request by nathan-barrett**
## Description
Created variation page for the /firefox/browsers/mobile/focus page, with variation `pb`. Baring issues, this should be live tomorrow 4/12
## Issue / Bugzilla link
#11464 
## Testing
https://localhost:8000/firefox/browsers/mobile/focus?v=pb  - and compare with base page. The QR code at the top right and adjust links for iOS and android mobile stores should be different between the two.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/browsers/mobile/focus.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
 {% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
 {% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
 {% set android_url = focus_adjust_url('android', 'mobile-focus-page') %}
+{% set variation_android_adjust_url = 'https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus-omc171-uk&redirect=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dorg.mozilla.focus' %}
+{% set variation_ios_adjust_url = 'https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus ![qr-code](https://user-images.githubusercontent.com/2030656/162119789-a4541259-76b6-4096-bddf-61867cb46f6e.svg) -omc171-uk&redirect=https%3A%2F%2Fapps.apple.com%2Fus%2Fapp%2Ffirefox-focus-privacy-browser%2Fid1055677337' %}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/browsers/mobile/focus.html:21]: This URL seems malformed? Maybe a wayward paste in the issue. 
```suggestion
{% set variation_ios_adjust_url = 'https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus-omc171-uk&redirect=https%3A%2F%2Fapps.apple.com%2Fus%2Fapp%2Ffirefox-focus-privacy-browser%2Fid1055677337' %}
```

File: bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-focus.html [hunk 2]
```diff
@@ -10,10 +10,14 @@
   <h2 class="c-inline-form-title">{{ ftl('mobile-focus-get-firefox-focus') }}</h2>
   <p class="c-inline-form-body">{{ ftl('mobile-focus-scan-the-qr-code-to-get-started') }}</p>
   <div class="qr-code-wrapper">
-  {% if LANG == 'de' %}
-    <img src="{{ static('img/firefox/browsers/mobile/qr-klar.png') }}" id="focus-qr" width="400" height="400" alt="{{ ftl('mobile-focus-scan-alt-text') }}">
+  {% if variation == 'pb' %}
+    <img src="{{ static('img/firefox/browsers/mobile/qr-variation.svg') }}"id="focus-qr" width="400" height="400" alt="{{ ftl('mobile-focus-scan-alt-text') }}" />
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-focus.html:14]: Missing a space between attributes, though the browser seems to cope.
```suggestion
    <img src="{{ static('img/firefox/browsers/mobile/qr-variation.svg') }}" id="focus-qr" width="400" height="400" alt="{{ ftl('mobile-focus-scan-alt-text') }}" />
```

**Review by craigcook — CHANGES_REQUESTED** [review]

File: bedrock/firefox/templates/firefox/browsers/mobile/focus.html (hunk 1 above)
Comment (nathan-barrett) [reply bedrock/firefox/templates/firefox/browsers/mobile/focus.html:21]: ah good catch, i'l double check with Dan before its ready

**Review by craigcook — APPROVED** [review]
LGTM! 🥜

**Pull request merged**